sigs.yaml

```
@@ -1425,15 +1425,15 @@ sigs:
     time: "7:30"
     tz: PT (Pacific Time)
     frequency: weekly
-    url: https://docs.google.com/document/d/1FQx0BPlkkl1Bn0c9ocVBxYIKojpmrS1CFP5h0DI68AE/edit
+    url: no-zoom-link
```

```suggestion
    url: https://zoom.us/my/k8s.mc
```

ref: https://docs.google.com/document/d/1v-Kb1pUs3ww_x0MiKtgcyTXCAuZlbVlz4_A9wS3_HXY/edit#